setup.cfg

```diff
@@ -0,0 +1,15 @@
+[flake8]
+show-source = True
+ignore = E203, E731, W503
+max-line-length = 88
```

Comment:
Here is a line length of 88, but in pyproject.toml a line length of 80 is specified.

Is this intended? Maybe line length for black is not a hard limit?

Or was this accidental?

Reply:
As explained to me in the other project, this seems to be best practice and even necessary as the value is no hard limit for black.